pkg/cache/v2/simple_delta.go

```diff
@@ -0,0 +1,297 @@
+// Copyright 2018 Envoyproxy Authors
```

Comment:
Curious if entirely new files shouldn't have the correct copyright year on them.